ansible/roles/providers/files/cloud-init-vmware.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
```

Comment:
Will users still be able to have their own cloud-init?

Reply:
Theoretically yes. However it would require an extra setup outside our normal flow : Create template using KIB -> Extra steps: (convert template to VM, change VM config go add userdata using UI or govc CLI or API), convert vm back to template) -> reference template name in DKP CLI when creating cluster. 
govc command: https://github.com/mevansam/vsphere-govmomi/blob/master/govc/USAGE.md#vmchange
```
   govc vm.change -vm "${VM}" \
  -e guestinfo.metadata="${METADATA}" \
  -e guestinfo.metadata.encoding="gzip+base64" \
  -e guestinfo.userdata="${USERDATA}" \
  -e guestinfo.userdata.encoding="gzip+base64"
```
The cluster-api uses it to configure VM using data from `KubeadmConfigTemplate`
https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/blob/master/pkg/services/govmomi/extra/config.go#L42